frontend/src/components/cluster/Charts.tsx

```diff
@@ -7,6 +7,7 @@ import { PercentageCircle, PercentageCircleProps } from '../common/Chart';
 
 interface ResourceCircularChartProps extends Omit<PercentageCircleProps, 'data'> {
   items: KubeObjectInterface[] | null;
+  metricsError: Error | null;
```

Comment:
This should probably just be called `error` since we don't do anything special about metrics, and maybe we can use the charts for things beyond displaying metrics. i.e. charts are agnostic to metrics, they're just used to display it, so we can call this `error`.